spec/factories/users.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    email "johndoe@hotmail.com"
```

Comment:
Create a sequence for generating emails. See this example https://til.hashrocket.com/posts/5a10f2bb1a-factorygirl-sequences